docker-compose-starburst.yml

```diff
@@ -16,45 +16,13 @@ services:
       POSTGRES_USER: dbt-trino
       POSTGRES_PASSWORD: dbt-trino
 
-  metastore_db:
-    image: postgres:11
-    hostname: metastore_db
-    environment:
-      POSTGRES_USER: hive
-      POSTGRES_PASSWORD: hive
-      POSTGRES_DB: metastore
-
   hive-metastore:
+    image: ghcr.io/innoverio/hive-metastore-multiarch:sha-28c28bc #TODO: replace with Starburst hive container once config is merged
```

Comment:
> replace with Starburst hive container once config is merged

Could you please use a concrete ticket in the  `TODO`  ? For me it is not clear at the moment what is the limitation of Starburst HMS  which requires the usage of a different HMS image.

Reply:
We need to discuss if we want it or not, the Starburst HMS doesn't offer direct access to the Hive config, so we would need to add environment variables and convert them to valid Hive configurations.

We also would need to consider if we want the Starburst HMS to support these exotic Hive settings.

Reply:
If I am not wrong, it's only `hive.users.in.admin.role` parameter which is not exposed in Starburst HMS, it shouldn't be a big deal to expose it there. Are there any other parameters which are not available in Starburst HMS?

Reply:
No other paremeters are required. I'll do a PR there but let that not stop us from releasing this one.